crates/zkvm/zisk/src/zkvm/sdk.rs

```diff
@@ -518,6 +538,38 @@ fn rom_setup(elf_path: &Path) -> Result<RomDigest, Error> {
     Ok(rom_digest)
 }
 
+/// Send shutdown request to ZisK asm services.
+fn shutdown_asm_service(port: u16) {
```

Comment:
When ZisK server crashes, the asm services are not cleanup properly, here we do it manually and it seems to resolves most of the server initialization halting issue